instrumentation/src/workflows/interceptors.ts

```diff
@@ -0,0 +1,34 @@
+// @@@SNIPSTART typescript-workflow-logging-interceptor
+import {
+  WorkflowInterceptorsFactory,
+  WorkflowInboundCallsInterceptor,
+  WorkflowExecuteInput,
+  Next,
+} from '@temporalio/workflow';
+import { logger } from './logger';
+
+/** Logs Workflow executions and their duration */
+export class WorkflowInboundLogInterceptor implements WorkflowInboundCallsInterceptor {
```

Comment:
Does this need to be exported?

Reply:
Removed the export so people don't get confused.